cpp/dolfinx/io/VTKWriter.cpp

```diff
@@ -180,8 +140,9 @@ void write_ascii_mesh(const mesh::Mesh& mesh, int cell_dim,
                       std::string filename)
 {
   const int num_cells = mesh.topology().index_map(cell_dim)->size_local();
-  const int degree = get_degree(mesh.geometry().cmap().cell_shape(),
-                                mesh.geometry().cmap().dof_layout().num_dofs());
+  const int degree
+      = io::cells::cell_degree(mesh.geometry().cmap().cell_shape(),
```

Comment:
This function should be private, i.e. in an anonymous namespace.

Reply:
Why? This function is also used in: https://github.com/FEniCS/dolfinx/blob/master/cpp/dolfinx/io/cells.cpp#L116

Reply:
Because it's not well defined, and should not be exposed. We don't care about the degree - we just need a VTK element string.

The `CoordinateMap` is what defines the cell geometry map.